dts/arm/nxp/nxp_s32z27x_r52.dtsi

```diff
@@ -103,103 +103,90 @@
 			compatible = "nxp,s32-linflexd";
 			reg = <0x40170000 0x1000>;
 			interrupts = <GIC_SPI 212 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
-			label = "UART_0";
```

Comment:
please amend initial commit

Reply:
amended the wrong commit before :) thanks